proposals/extensions.md

```diff
@@ -693,7 +693,11 @@ TODO4 need to merge extension members and extension methods
     - Merge the results
     - Next, members that are hidden by other members are removed from the set.  
       (note: "base types" means "base extensions and underlying type" for extension types)
-    - Finally, having removed hidden members:
+    - Next, extension members are removed if they are "hidden" by more specific extension members. 
+      For every member `X.M` in the set, where `X` is the type in which the member `M` is declared, the following rules are applied:
```

Comment:
I think there is a deeper problem :-/ 
These rules require a specific iteration order. Otherwise, in a "method hiding a field hiding a method" situation, we could end up removing the last method, which is wrong.
I'll take another stab

Reply:
Note, I think this spec problem is already present in the [current member lookup rules](https://github.com/dotnet/csharpstandard/blob/draft-v8/standard/expressions.md#125-member-lookup)...

Reply:
> These rules require a specific iteration order. Otherwise, in a "method hiding a field hiding a method" situation, we could end up removing the last method, which is wrong.

I do not think the suggested wording has this problem. We can discuss offline.

>Note, I think this spec problem is already present in the current member lookup rules...

I don't think this helps us in any way. We need to specify clear rules so that they can be reviewed, clearly understood by others, and then approved or rejected at design meeting.